presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestBinaryStatisticsBuilder.java

```diff
@@ -53,24 +55,11 @@ public void testSum()
     {
         BinaryStatisticsBuilder binaryStatisticsBuilder = new BinaryStatisticsBuilder();
         for (Slice value : ImmutableList.of(EMPTY_SLICE, FIRST_VALUE, SECOND_VALUE)) {
-            binaryStatisticsBuilder.addValue(value);
+            addValue(binaryStatisticsBuilder, value);
         }
         assertBinaryStatistics(binaryStatisticsBuilder.buildColumnStatistics(), 3, EMPTY_SLICE.length() + FIRST_VALUE.length() + SECOND_VALUE.length());
     }
 
-    @Test
-    public void testSliceWithIndexLength()
-    {
-        BinaryStatisticsBuilder binaryStatisticsBuilder = new BinaryStatisticsBuilder();
-        Slice slice = utf8Slice("abcdefghijklmnopqrstuvwxyz");
-        for (int i = 0; i < slice.length(); i++) {
-            binaryStatisticsBuilder.addValue(slice, i, 1);
-        }
-
-        BinaryStatistics binaryStatistics = binaryStatisticsBuilder.buildColumnStatistics().getBinaryStatistics();
-        assertEquals(binaryStatistics.getSum(), slice.length());
-    }
-
```

Comment:
Shall we add something similar for block ? also in TestStringStatisticsBuilder ?